protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMailboxProcessor.java

```diff
@@ -100,6 +101,10 @@ protected final Mono<Void> doProcess(R acceptableMessage, Responder responder, I
                             no(acceptableMessage, responder, HumanReadableText.DENIED_SHARED_MAILBOX);
                             return Mono.empty();
                         })
+                        .onErrorResume(OverQuotaException.class, e -> {
+                            no(acceptableMessage, responder, HumanReadableText.FAILURE_OVERQUOTA);
+                            return Mono.empty();
```

Comment:
Maybe an INFO log could be a good idea?

Reply:
BTW we can add an overQuota response code, cf https://datatracker.ietf.org/doc/html/rfc5530

Quote from `AppendProcessor`: `no(request, responder, HumanReadableText.FAILURE_OVERQUOTA, StatusResponse.ResponseCode.overQuota());`